t/unit/utils/test_platforms.py

```diff
@@ -659,6 +659,20 @@ def test_remove_if_stale_broken_pid(self):
             assert p.remove_if_stale()
             p.remove.assert_called_with()
 
+    @patch('os.kill')
+    def test_remove_if_stale_unprivileged_user(self):
+        with mock.stdouts():
+            p = Pidfile('/var/pid')
+            p.read_pid = Mock()
+            p.read_pid.return_value = 1817
+            p.remove = Mock()
+            exc = OSError()
+            exc.errno = errno.EPERM
+            kill.side_effect = exc
```

Comment:
where this `kill` comes from? 
Normaly mocks performed through `@patch` are passed as argument to the def test method.

Reply:
Sorry.  Typo in first commit.